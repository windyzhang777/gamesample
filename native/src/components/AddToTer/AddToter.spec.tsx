import React from 'react';
import {shallow} from 'enzyme';
// import AddToter from './index';

describe('<AddToter/>', () => {
  it('should remove this passing test', () => {
    expect(true).toBeTruthy();
  });
  // it('renders wrapper', () => {
  //   expect(shallow(<AddToter />)).toMatchSnapshot();
  // });
  // it('renders component', () => {
  //   const RenderPropComponent = shallow(<AddToter />).prop('renderComponent');
  //   expect(
  //     shallow(
  //       <RenderPropComponent
  //         changeToterNameCallback={jest
  //           .fn()
  //           .mockName('A function to be called when the user types a keystroke')}
  //         addToterCallback={jest
  //           .fn()
  //           .mockName('function to be called when the add Toter button is tapped')}
  //         username={'gmom-35'}
  //       />,
  //     ),
  //   ).toMatchSnapshot();
  // });
});
