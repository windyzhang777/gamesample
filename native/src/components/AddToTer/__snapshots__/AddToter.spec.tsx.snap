// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddToter/> renders component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#5c446f",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
        "margin": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "flex": 1,
          "fontWeight": "bold",
        }
      }
    >
      Trick or Treater Name
    </Text>
    <Component
      allowFontScaling={true}
      onChangeText={[MockFunction A function to be called when the user types a keystroke]}
      rejectResponderTermination={true}
      style={
        Object {
          "backgroundColor": "white",
          "borderRadius": 20,
          "flex": 1,
          "paddingHorizontal": 20,
          "paddingVertical": 5,
        }
      }
      underlineColorAndroid="transparent"
      value="gmom-35"
    />
  </View>
  <ForwardRef
    onPress={[Function]}
  >
    <Text
      style={
        Object {
          "borderColor": "white",
          "borderRadius": 20,
          "borderWidth": 1,
          "color": "white",
          "marginTop": 20,
          "padding": 8,
          "paddingHorizontal": 20,
          "textTransform": "capitalize",
        }
      }
    >
      Add a new ToTer
    </Text>
  </ForwardRef>
</View>
`;

exports[`<AddToter/> renders wrapper 1`] = `
<AddToter
  renderComponent={[Function]}
/>
`;
